frontend/src/scenes/onboarding/personalizationLogic.ts

```diff
@@ -32,7 +32,7 @@ export const personalizationLogic = kea<personalizationLogicType<Personalization
         },
         reportPersonalization: async ({ payload, step_completed_fully }) => {
             posthog.people.set_once(payload)
-            posthog.capture('personalization completed', {
+            posthog.capture('personalization finalized', {
```

Comment:
won't this cause pain in analysis due to dropping old data?

Reply:
Not sure I follow, basically we are reporting when the user clicks continue on the personalization screen (whether they have actually answered all questions or not), and we pass `step_completed_fully` to know if questions were answered in full or not.

Reply:
Since the old name changes wouldn't we essentially drop the current data?

Reply:
Oh got it! Well the new onboarding hasn't been released to anyone yet so we're good